src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -665,6 +665,16 @@ private StorageEnvironmentOptions CreateStorageEnvironmentOptions(DocumentDataba
 
             var indexTempPath = configuration.TempPath?.Combine(name);
 
+            if (configuration.RunInMemory == false)
+            {
+                string disableMarkerPath = indexPath.Combine("disable.marker").FullPath;
+                if (File.Exists(disableMarkerPath))
+                {
+                    throw new IndexOpenException("Unable to open index: " + name + ", it has been manually disabled via the file: '" + disableMarkerPath +"'." +
+                                                 "To re-enable, remove the disable.marker file and enable indexing.");
```

Comment:
This will create a faulty index, right? After removing the file, how we can reopen the index without reloading everything?

Reply:
You have:
![image](https://user-images.githubusercontent.com/116915/184095975-80fe31af-ef6b-439f-931a-60d31ec96ef7.png)

Remove the disable marker and click:

![image](https://user-images.githubusercontent.com/116915/184096081-f06c18bb-5326-4ee0-9302-a4e6af22088f.png)

You have 💮 

![image](https://user-images.githubusercontent.com/116915/184096120-3793d747-8c56-4861-b55a-2ca3524b9f56.png)